### ISSUE TITLE
could not start language server on julia 1.3.1 windows 10

### ISSUE THREAD

**Issue by melspectrum** [julia-vscode/julia-vscode#999]
fresh install vscode Version: 1.42.1 and julia 1.3.1

updated settings to the new install

`"julia.executablePath": "D:\\Julia-1.3.1\\bin\\julia.exe"`

vscode reported `could not start language server`

**Comment by davidanthoff** [comment]
Is D: a normal local disc? Or some network drive? If you hit WinKey + R and then paste that string in there, does that open Julia just fine?

**Comment by melspectrum** [comment]
Yes, I just tried it. It opened Julia just fine, with '\\' in the path.

**Comment by davidanthoff** [comment]
What about "Is D: a normal local disc? Or some network drive?"?

**Comment by melspectrum** [comment]
oh, it is a normal local disc. Is it a possible permission issue? I have another computer with default julia installation options (under C:\USERS\${USER}) and everything works fine.

**Comment by davidanthoff** [comment]
Maybe, although it is not really clear to me why this wouldn't work... I don't really have a great idea right now what the problem might be here...

**Comment by davidanthoff** [comment]
Closing as outdated: the language server now requires Julia 1.11 or newer (it refuses to start below that), so the configuration in this report is no longer supported. Juliaup, environment discovery and language server startup have all been reworked since as well.

If you hit something similar on a current setup, please open a new issue with the details and we will take a look.
